Source/Scene/Cesium3DTile.js

```diff
@@ -426,10 +409,13 @@ define([
      */
     Cesium3DTile.prototype.destroy = function() {
         this._content = this._content && this._content.destroy();
-        this._debugBox = this._debugBox && this._debugBox.destroy();
-        this._debugcontentBox = this._debugcontentBox && this._debugcontentBox.destroy();
-        this._debugOrientedBoundingBox = this._debugOrientedBoundingBox && this._debugOrientedBoundingBox.destroy();
-        this._debugContentsOrientedBoundingBox = this._debugContentsOrientedBoundingBox && this._debugContentsOrientedBoundingBox.destroy();
+        // TODO: Is there a reason to destroy the bounding volumes (and thus to implement destroy() functions for them)? The destroyObject function description says it should be used to for objects that have GL resources. 
```

Comment:
This is only needed for the `debug` volumes since they contain Cesium primitives, which contain GL resources.